src/utils/document_validator.py

```
@@ -0,0 +1,79 @@
+        ".docx",
+        ".rtf",
+        ".html",
+        ".htm",
+        ".xml",
+        ".json",
+        ".csv",
+        ".xlsx",
+        ".xls",
+        ".pptx",
+        ".ppt",
+    }
```

Ruff (0.14.10)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)